moveit_cpp:
  # This tells our Python script to force Sim Time
  use_sim_time: true
  
planning_scene_monitor_options:
  name: "planning_scene_monitor"
  robot_description: "robot_description"
  joint_state_topic: "/joint_states"
  attached_collision_object_topic: "/attached_collision_object"
  publish_planning_scene_topic: "/publish_planning_scene"
  monitored_planning_scene_topic: "/monitored_planning_scene"
  wait_for_initial_state_timeout: 10.0

planning_pipelines:
  pipeline_names:
    - ompl
    - pilz_industrial_motion_planner  # <--- 1. ADD THIS

plan_request_params:
  planning_attempts: 1
  planning_pipeline: ompl
  max_velocity_scaling_factor: 1.0
  max_acceleration_scaling_factor: 1.0

pilz_industrial_motion_planner:
  planning_plugin: "pilz_industrial_motion_planner/CommandPlanner"
  request_adapters: ""
  default_planner_config: "PTP"